static/src/javascripts/projects/commercial/modules/dfp/define-slot.js

```diff
@@ -38,8 +37,10 @@ const buildSizeMapping = (sizes) => {
 const getSizeOpts = (sizesByBreakpoint) => {
     const sizeMapping = buildSizeMapping(sizesByBreakpoint);
     // as we're using sizeMapping, pull out all the ad sizes, as an array of arrays
+
+    const flattenSizeMappings = sizeMapping.map(size => size[1]).reduce((a, b) => [...a, ...b], [])
```

Comment:
I think this should only be a single level deep, and `b` should be returned as-is.

This also leads to only a [7% slowdown][jsbench] compared to Lodash.

[jsbench]: https://jsbench.me/jlkmut9o3y/1

```suggestion
    const flattenSizeMappings = sizeMapping.map(size => size[1]).reduce((a, b) => [...a, b], [])
```

Reply:
That gives incorrect result as the array is not flatten.  
I've switched to spread operator because [it's faster than concat] (https://jsbench.me/tqkmvvw76s/1).

Reply:
Updated performance results with spread, almost equal perf https://jsbench.me/sckmuq8pk7/1

Reply:
You’re right, but there’s still an issue, when `b` isn’t iterable.

https://replit.com/@mxdvl/Flatten <- might be worth capturing this in a test.